### PR TITLE
Feature/cu p3d1rc new lose condition

### PR DESCRIPTION
**Pull request by devildrake**
Established lose condition to be both characters being dead at the same time, made some slight UI code modifications to match what design wanted, established forceful character switch on character death.

POSSIBLE PROBLEM: We must make sure that the static bool inside gamemanager regardin the ability to switch to onimaru is true in levels that are not level 1, otherwise one character dying will mean game over

### CODE REVIEW

**Comment by jcarlos0305** [comment]
Task linked: [CU-p3d1rc Change lose condition](https://app.clickup.com/t/p3d1rc)

File: Penteract/Assets/Scripts/PlayerController.cpp [hunk 1]
```diff
@@ -368,27 +404,27 @@ void PlayerController::Update() {
 		playerOnimaru.Update(useGamepad);
 	}
 
-	if (playerFang.isAlive && playerOnimaru.isAlive) {
-		CheckCoolDowns();
-		UpdatePlayerStats();
+	//if (playerFang.isAlive && playerOnimaru.isAlive) {
```
Comment (FrancPS) [inline Penteract/Assets/Scripts/PlayerController.cpp:407]: this line should be commented?
If ins not necessary, should it be removed?

**Review by FrancPS — APPROVED** [review]

File: Penteract/Assets/Scripts/Fang.cpp [hunk 2]
```diff
@@ -239,23 +239,26 @@ bool Fang::CanEMP() {
 }
 
 void Fang::CheckCoolDowns(bool noCooldownMode) {
-	if (!fangLaser) return;
+	//if (!fangLaser) return;
```
Comment (magalenyo) [inline Penteract/Assets/Scripts/Fang.cpp:242]: Undo this change

Comment (devildrake) [reply Penteract/Assets/Scripts/Fang.cpp:242]: Done

File: Penteract/Assets/Scripts/PlayerController.cpp (hunk 1 above)
Comment (devildrake) [reply Penteract/Assets/Scripts/PlayerController.cpp:407]: You're absolutely right

**Review by JaimeBea — APPROVED** [review]

**Review by FrancPS — APPROVED** [review]

**Review by jcarlos0305 — DISMISSED** [review]

**Review by isaacfdez — APPROVED** [review]

**Review by FrancPS — DISMISSED** [review]

**Review by isaacfdez — APPROVED** [review]

**Review by FrancPS — DISMISSED** [review]

**Review by Mefiso — APPROVED** [review]

**Review by JaimeBea — DISMISSED** [review]

**Review by Mefiso — APPROVED** [review]

**Review by FrancPS — APPROVED** [review]

**Review by isaacfdez — APPROVED** [review]
